#ifndef NES_INCLUDE_GRPC_COORDINATORRPCSERVER_HPP_
#define NES_INCLUDE_GRPC_COORDINATORRPCSERVER_HPP_

#include <iostream>
#include <memory>
#include <string>
#include <GRPC/CoordinatorRPCServerImpl.hpp>

class CoordinatorRPCServer {
  public:

    CoordinatorRPCServer();

};

#endif //NES_INCLUDE_GRPC_COORDINATORRPCSERVER_HPP_
